From cb621aad645445ef20e771a6a302d8cf30be8c63 Mon Sep 17 00:00:00 2001
From: Leonid Komarianskyi <leonid_komarianskyi@epam.com>
Date: Mon, 27 May 2024 10:18:43 +0300
Subject: [PATCH 3/4] gen5: vdk: add proper 10GB of RAM region

According to information from Renesas, 10GB region of RAM
started from 0x1080000000 is available in VDK 3.6.3.
Previous regions are mapped to nowhere, and can't be used,
so they should be removed.

Signed-off-by: Leonid Komarianskyi <leonid_komarianskyi@epam.com>
---
 arch/arm/dts/rcar-x5h-board-u-boot.dts | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/arch/arm/dts/rcar-x5h-board-u-boot.dts b/arch/arm/dts/rcar-x5h-board-u-boot.dts
index 28ca6d336c..1c1761dc28 100644
--- a/arch/arm/dts/rcar-x5h-board-u-boot.dts
+++ b/arch/arm/dts/rcar-x5h-board-u-boot.dts
@@ -29,19 +29,9 @@
 		reg = <0x0 0x48000000 0x0 0x78000000>;
 	};
 
-	memory@480000000 {
+	memory@1080000000 {
 		device_type = "memory";
-		reg = <0x4 0x80000000 0x0 0x80000000>;
-	};
-
-	memory@800000000 {
-		device_type = "memory";
-		reg = <0x8 0x00000000 0x1 0x00000000>;
-	};
-
-	memory@c00000000 {
-		device_type = "memory";
-		reg = <0xc 0x00000000 0x1 0x00000000>;
+		reg = <0x10 0x80000000 0x2 0x80000000>;
 	};
 };
 
-- 
2.25.1

